src/python.cc

```diff
@@ -208,21 +208,24 @@ class ModelInstanceState : public BackendModelInstance {
       ModelState* model_state, TRITONBACKEND_ModelInstance* model_instance);
 
   TRITONBACKEND_Model* triton_model_;
-  pthread_mutex_t* child_mutex_;
-  pthread_cond_t* child_cond_;
+  pthread_mutex_t* stub_mutex_;
+  pthread_cond_t* stub_cond_;
   pthread_mutex_t* parent_mutex_;
   pthread_cond_t* parent_cond_;
   std::string model_path_;
   IPCMessage* ipc_message_;
   std::unique_ptr<SharedMemory> shm_pool_;
-  // Child process pid
-  pid_t child_pid_;
+  // Stub process pid
+  pid_t stub_pid_;
 
   // Parent process pid
   pid_t parent_pid_;
-
```

Comment:
Keep this empty line

Reply:
I think we should only have empty lines before the comments so I think the removal is ok here.